src/SetChunkData.h

```diff
@@ -51,20 +49,11 @@ class cSetChunkData
 	int GetChunkX(void) const { return m_ChunkX; }
 	int GetChunkZ(void) const { return m_ChunkZ; }
 
-	/** Returns the internal storage of the block types, read-only. */
-	const cChunkDef::BlockTypes & GetBlockTypes(void) const { return m_BlockTypes; }
-
-	/** Returns the internal storage of the block types, read-only. */
-	const cChunkDef::BlockNibbles & GetBlockMetas(void) const { return m_BlockMetas; }
-
-	/** Returns the internal storage of the block light, read-only. */
-	const cChunkDef::BlockNibbles & GetBlockLight(void) const { return m_BlockLight; }
-
-	/** Returns the internal storage of the block types, read-only. */
-	const cChunkDef::BlockNibbles & GetSkyLight(void) const { return m_SkyLight; }
+	/** Returns the internal storage of block types, metas and lighting. */
+	cChunkData & GetChunkData(void) { return m_ChunkData; }
 
 	/** Returns the internal storage for heightmap, read-only. */
-	const cChunkDef::HeightMap & GetHeightMap(void) const { return m_HeightMap; }
+	cChunkDef::HeightMap & GetHeightMap(void) { return m_HeightMap; }
```

Comment:
The comment no longer matches the function (non-const)

Reply:
Sorry, stale change.  I've fixed it now.